packages/manager/modules/pci/src/projects/project/storages/databases/database/query-statistics/query-statistics.controller.js

```diff
@@ -0,0 +1,68 @@
+export default class {
+  /* @ngInject */
+  constructor($translate, CucCloudMessage, DatabaseService) {
+    this.$translate = $translate;
+    this.CucCloudMessage = CucCloudMessage;
+    this.DatabaseService = DatabaseService;
+  }
+
+  $onInit() {
+    this.loadMessages();
+    this.trackDashboard('query_statistics', 'page');
+    this.messageContainer =
```

Comment:
Define the messageContainer variable before calling the loadMessages since it's used in the function